### ISSUE TITLE
Jetpack Connection: Allow users to select a different account to sign in with even if there's an email match

### ISSUE THREAD

**Issue by chaselivingston** [Automattic/wp-calypso#21655]
If there’s a WordPress.com account linked to your self-hosted account’s email address, there doesn’t seem to be a way to choose to create a new account without exiting the Jetpack connection flow. I discovered this because when you create a poopy.life site, you’re logged in to the site with a demo user with the email address demo@soflyy.com, which is connected to a WordPress.com account. After selecting “Connect with WordPress.com” I was stuck on this login screen:

![image](https://user-images.githubusercontent.com/831377/35112000-b4294eee-fc4a-11e7-9b30-e15f138e6055.png)

The only option I saw to create a new account was “Sign Up” in the top right, which took me out of the Jetpack connection flow and into the regular WordPress.com user+site signup. Although the poopy.life case is very specific, this could happen to anyone who wants to create a different WordPress.com account for their new Jetpack site.

**Comment by joanrho** [comment]
Hey @chaselivingston! This was discussed this a bit more here: p1HpG7-4Mw/#comment-24585.
TL;DR: to start, we'll fix the masterbar link to point to the correct account creation form and gather data on that change.

**Comment by beaulebens** [comment]
Closing this since I believe it was fixed in #22488, and will be further improved with some coming changes.

**Comment by rickybanister** [comment]
What if we did something a bit more radical here and just made an exception for the poopy.life email address since we know it's a shared address? I'm not sure if this is a real world issue.

**Comment by rachelmcr** [comment]
>I'm not sure if this is a real world issue.

Just noting that this has come up again in https://github.com/Automattic/wp-calypso/issues/24127 (a real world case).
